exporter/elasticsearchexporter/pdata_serializer.go

```diff
@@ -331,7 +434,10 @@ func writeValue(v *json.Visitor, val pcommon.Value, stringifyMaps bool) {
 
 func writeTimestampField(v *json.Visitor, key string, timestamp pcommon.Timestamp) {
 	_ = v.OnKey(key)
-	_ = v.OnString(timestamp.AsTime().UTC().Format(tsLayout))
+	nsec := uint64(timestamp)
```

Comment:
nit: changing timestamp to a float may make it harder to understand the `telemetry::log_request_body` output, but if it gets us better performance in most cases, it is fine by me

Reply:
That's fair, but I think the performance gains both here and in ES parsing the timestamp, are worth it.